### ISSUE TITLE
Compatibility with latest ASF release

### ISSUE THREAD

**Issue by TomasMFranco** [alvr/ASFui#25]
Things like Start all never worked for me.
The GUI can't parse the request for 2FA codes.
But now the buttons are no longer doing anything.

**Comment by thunderpants** [comment]
yep ,nothing works with asf3.1.1.9  ,redeem ,status, add license e.t.c ,e.t.c.

Not even sure if it's even farming cards. 
it does seem to login after redoing all my account configs.

getting 404 "address not found" errors  and 405 method not allowed errors

Would be nice if someone with the skills could takeover this software and fix the problems.

**Comment by Luckz** [comment]
New visitors might want to know that there are three implementations of ASFui using three different programming languages (C#/WPF; Kotlin/JavaFX; Python/Qt), and especially for novice Github users it might be unintuitive to check https://github.com/alvr/ASFui/network (and scroll back really far to the left, and then to the right again) to see if any of them might actually have gotten some of those problems fixed. Spoilers: it's the first / oldest one. The other two shouldn't really be functional at present.

There have definitely always™ been some working versions, so far. I wouldn't expect all functionality to work incredibly well though, as none of the people who have worked on it in the past seem interested in spending much time on maintaining pristine conditions. Being able to activate keys, use it as 2FA client, perhaps farm some cards and confirm trades is _good enough_.
